.sikkerhet/beskrivelse.yaml

```diff
@@ -0,0 +1,5 @@
+version: 2.0
+organization: IT
+product:
```

Comment:
Dette området inneholder dokumenter på html-format.
Dokumentene er arbeidsversjoner av produktspesifikasjoner for fagområdet Arealplan. 
Området skal ikke inneholde programkode.